test/integration/config/setup.js

```diff
@@ -0,0 +1,10 @@
+require('@babel/register');
+require('ts-node').register({ transpileOnly: true });
+
+require('../../helpers/setup-helper');
+
+window.SVGPathElement = window.SVGPathElement || { prototype: {} };
+global.indexedDB = {};
```

Comment:
Nit: Much of the contents of this file are identical to `test/setup.js`. I wonder if the common steps should be moved into `setup-helper.js`? e.g. reserve this file for integration-test-specific setup, and `test/setup.js` for unit test specific setup.

Reply:
I suppose the same suggestion applies to `setupAfter` as well, except that there is no shared module for that yet.

Reply:
done 👍